tests/unit/test_ntm.py

```diff
@@ -286,11 +286,11 @@ def test_call_fit_wrong_type_mini_batch_size(sagemaker_session):
     data = RecordSet("s3://{}/{}".format(BUCKET_NAME, PREFIX), num_records=1, feature_dim=FEATURE_DIM,
                      channel='train')
 
-    with pytest.raises(ValueError):
+    with pytest.raises((TypeError, ValueError)):
```

Comment:
Some comments here: 

The reason I use tuple (TypeError, ValueError) is because, different error is raised in different python versions. Python 2 raises ValueError and Python 3 raises TypeError.